docs/index.rst

```diff
@@ -93,6 +93,23 @@ Using the API is done as follows::
         from celery import current_app
         current_app.backend.mark_as_done(request.id, response, request=request)
 
+.. note::
+
+    The use of ``current_app.backend.mark_as_retry()`` and ``current_app.backend.mark_as_failure()`` are not currently supported.
```

Comment:
`mark_as_failure` is used in the unit test, but is mentioned here as not supported -- can you elaborate on what this means then?

Reply:
You're right, I think I had left `mark_as_failure()` in there during testing. It can be removed, and the new unit tests still pass, so I'll remove it.

Reply:
I think this comment is still outstanding?

Reply:
Addressed in this commit: https://github.com/clokep/celery-batches/pull/59/commits/08a7a1bad637fb3e0b62386cdc3a7a7c7f7ee0b3